#! /bin/sh /usr/share/dpatch/dpatch-run

## 01_dist-var.dpatch by Thomas Schmidt <thomas.schmidt@in.stud.tu-ilmenau.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change vdradmin.pl to search for it's files in system-directories,
## DP: instead of local, relative paths

@DPATCH@
diff -urNad /home/chelli/vdr/cvs/vdr/vdradmin/vdradmind.pl vdradmin/vdradmind.pl
--- /home/chelli/vdr/cvs/vdr/vdradmin/vdradmind.pl	2004-05-27 16:34:55.000000000 +0200
+++ vdradmin/vdradmind.pl	2004-05-27 16:35:55.000000000 +0200
@@ -42,7 +42,7 @@
 #use warnings;
 
 #my $SEARCH_FILES_IN_SYSTEM = (-d '/usr/share/vdradmin/lib' ? 1 : 0); # for distribution
-my $SEARCH_FILES_IN_SYSTEM = 0;
+my $SEARCH_FILES_IN_SYSTEM = 1;
 
 sub true           () { 1 };
 sub false          () { 0 };
